.tests/misc-consistent-linked-issues

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import re
+import sys
+import textwrap
+from github import Github
+
+# Regex to pick out closing keywords.
+LINKED_ISSUES = re.compile("(close[sd]?|fix|fixe[sd]?|resolve[sd]?)\s*:?\s+#(\d+)", re.I)
+
+# Returns a pull request extracted from Github's event JSON.
+def get_pr(event):
+    # --- Extract issue from event JSON ---
+    # `pull_request` directly refers the the PR number in the event JSON.
+    # Extract that number and use it to grab the PR.
+    pr_number = event["number"]
+
+    # Grab the PR from the number and return
+    return repo.get_pull(pr_number)
+
+# Extract all associated issues from PR commit messages
+def get_linked_issues_commits(pr):
+    for c in pr.get_commits():
+        for verb, num in LINKED_ISSUES.findall(c.commit.message):
+            yield num
+
+# Extract all associated issues linked in PR description
+def get_linked_issues_body(pr):
+    # Extract all associated issues from closing keyword in PR
+    for verb, num in LINKED_ISSUES.findall(pr.body):
+        yield num
+
+# Get inputs from shell
+(token, repository, path) = sys.argv[1:4]
+
+# Initialize repo
+repo = Github(token).get_repo(repository)
+
+# Open Github event JSON
+with open(path) as f:
+    event = json.load(f)
+
+# Get the PR we're working on.
+pr = get_pr(event)
+
+commits_issues = set(get_linked_issues_commits(pr))
+description_issues = set(get_linked_issues_body(pr))
+unlinked_issues = commits_issues - description_issues
+
+if len(unlinked_issues) > 0:
+    header = "Your PR contains one or more commit messages that refer to closing issues, but those issues are not linked in your pull request description. Please include the following in your PR description:"
+    wrapper = textwrap.TextWrapper(width=80)
+    header_lines = wrapper.wrap(text=header)
+    for line in header_lines:
+        print(line)
+    print("")
```

Comment:
Now that I see it in action, using `textwrap` seems like overkill. Just split is manually, possibly using a multiline string for readability.